docs/tutorials/polybeam_simulation.ipynb

```diff
@@ -15,37 +15,26 @@
   },
```

Comment:
<p>Wishlist item (i.e., add as an issue, but don't address in this PR): demonstrate the effects of the perturbed beam through imaging.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Issue made: <a href="https://github.com/HERA-Team/hera_sim/issues/327" target="_blank">https://github.com/HERA-Team/hera_sim/issues/327</a></p>

Reply:
<p>Thanks!</p>